package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "tram-one",
-  "version": "10.1.5",
+  "version": "10.1.6",
```

Comment:
no API change, fixes an issue with a feature implementation, so patch is good :+1: